_posts/2016/2016-02-17-development-from-the-treches.md

```diff
@@ -0,0 +1,59 @@
+{% image trivia.jpg %}{% endimage %}
+
+### For developers
+As everybody knows Barcelona is one of the most important European cities in software development and hosts the Mobile World Congress, which brings amazing opportunities and projects to the city. This attracts talent which enriches the existing pool of [meet ups](http://www.meetup.com/cities/es/barcelona/) or technical events, where you can participate, learn or collaborate with your thoughts or ideas.
+
+## The office
+{% image bcn-barbecue.jpg 70% right %}{% endimage %}
+The office is located in the heart of Barcelona, in the Diagonal avenue, where it is full of shops, restaurants and good public transport connections. This excellent location offers us many different ways to come to the office in a decent time and choose what do you want to eat every day (Galician, French, Japanese or a mix of all of them!) or take away something to the office.
+The environment in the bureau is nice, right now we are 14 people from 4 different countries and 5 speaking languages, divided in 5 departments (admin, design, development, marketing, and sales).
+Every Friday morning we have the **tea time** where we all meet to talk about our 'things' and each team is responsible for presenting their last projects.
+
+### Events
+In regular basis we organise after hours activities like lunches, dinners or sailing days!!!
+the company organises other events such as the ePagesYou, which is the convention of the company and takes place in Hamburg, and the Hackathon, when all developers meet for 2 days to create apps for our appstore.
+
+## Team
+Our team consists of 4 engineers, we are looking for a [QA](http://www.epages.com/en/career/jobs/QA_BCN-HH.php) and two more [developers](http://www.epages.com/en/career/jobs/software-developer.php). The QA, Scrum Master and Product Owner are located in Hamburg and support us in our daily basis work.
+Our team is responsible, together with another team in Hamburg, for the connectors to the e-commerce platform, such as payments, shippings and others.
+
+Also, as in the company we try to do our best to be as **Agile** as possible, we follow the **Scrum** principles, we do **pair programming**, we share our knowledge in daily code reviews, we do presentations, etc.
+
+### How do we organise ourselves?
+Maybe you are wondering how do we manage to organise ourselves without a scrum master and still have time to take a coffee?
+One of us plays the role of **scrum master** as well as developer, but when things get crazy we have the Hamburg scrum master to helps us.
+
+{% image dev-bc-group2.jpg 50% left %}{% endimage %}
+The **product owner** is also placed in Hamburg but we are in touch with him almost every day because he attends our dailies, reviews and plannings. He also comes to Barcelona once a month to discuss the progress of the different projects or plan the new ones.
```

Comment:
The **product owner** is also placed in Hamburg but... > The **product owner** is also located in Hamburg, but we are in touch with him almost every day. He attends our dailies,..